SayTheirNames/Source/View/ImageWithBlurView.swift

```diff
@@ -66,9 +66,9 @@ class ImageWithBlurView: UIView {
     }
     
     // MARK: - Method
-    public func setup(_ person: Person) {
-        guard person.images.first != nil else { return }
-        frontImageView.populate(withURL: person.images[0].personURL)
-        bgImageView.populate(withURL: person.images[0].personURL)
+    public func setup(withURL urlString: String?) {
```

Comment:
small nitpick, label is withURL but you're passing a string...

Reply:
I don't think that's a problem, because that's a URL string. Do you mean to URL instead of String, or just to change the function's outside name? i.e. `public func setup(withURLString string: String?)`

Reply:
@dasautoooo yeah exactly just change the label to `withURLString`...nice one +1